sources/release-server/ReleaseServer.WebApi/Startup.cs

```diff
@@ -10,18 +19,23 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.OpenApi.Models;
+
 using Newtonsoft.Json;
-using ReleaseServer.WebApi.Auth;
-using ReleaseServer.WebApi.Config;
-using ReleaseServer.WebApi.Extensions;
-using ReleaseServer.WebApi.JsonConverters;
-using ReleaseServer.WebApi.SwaggerDocu;
+
 using Swashbuckle.AspNetCore.Filters;
 
 namespace ReleaseServer.WebApi
 {
-    public class Startup
+    internal class Startup
```

Comment:
Usually `Startup` is marked as `public`.

Reply:
Done with https://github.com/Traeger-GmbH/release-server/pull/60/commits/2cb28c7104c2450252ad2f226c26b9bc0480c9a1